definitions/npm/react-router-dom_v6.x.x/flow_v0.104.x-/test_react-router-dom.js

```diff
@@ -484,5 +485,54 @@ describe("react-router-dom", () => {
         sensitive: 'foo',
       });
     });
+
+    it('useMatches', () => {
+      type Params<Key: string> = {
```

Comment:
Import it from the definition?   
Might need an `export`.

Reply:
done